tutteli-gradle-spek/src/main/groovy/ch/tutteli/gradle/spek/SpekPlugin.groovy

```diff
@@ -78,4 +79,8 @@ class SpekPlugin implements Plugin<Project> {
             throw new IllegalStateException(ERR_KOTLIN_PLUGIN, e)
         }
     }
+
+    private static String getJunitVersion(Project project) {
```

Comment:
as far as I can see you are not using this method anywhere, please remove

Reply:
True, I found no way to create somthing similar to `getKotlinVersion`, unfortunately. Done.